src/serac/physics/state/state_manager.hpp

```diff
@@ -131,8 +142,18 @@ class StateManager {
   static std::string collection_name_;
   /**
    * @brief A set of @p QuadratureData<T> objects that need to be synchronized before saving to disk
+   * FIXME Need one set per datacoll since we can save at different times?
    */
   static std::vector<std::unique_ptr<SyncableData>> syncable_data_;
+
+  /// @brief Pointer (non-owning) to the datastore
+  static axom::sidre::DataStore* ds_;
+  /// @brief Output directory to which all datacollections are saved
+  static std::string output_dir_;
+  /// @brief Cycle to load - for restarts - FIXME can this be different between datacollections?
```

Comment:
Instead of keeping track of the cycle to load (set via a param to StateManager::initialize) I think it would be clearer to have a set of roughly symmetrical functions:
``cpp
StateManager::save(double t, int cycle, <datacoll tag>); // already exists
StateManager::load(int cycle, <datacoll tag>);
```

This provides additional user flexibility when needed and also eliminates the need to somehow restore what datacollections need to be restored (their names, specifically) before restoring the datacollections.

Reply:
Apologies for not fully understanding the context - would this be loading a different place within a solver than one was at previously? I could see this being one possible way to support adjoint analysis of transient problems, if "cycle" can be taken to be synonymous with "timestep."